improver_tests/utilities/test_OccurrenceWithinVicinity.py

```diff
@@ -221,6 +157,16 @@ class Test_process(IrisTest):
     def setUp(self):
         """Set up distance."""
         self.distance = 2000
+        coords = np.array([0.0, 2000.0, 4000.0, 6000.0])
+        self.timesteps = np.array([402192.5, 402195.5], dtype=np.float32)
```

Comment:
We should specify times in seconds whilst we are updating things.
```suggestion
        coords = np.array([0.0, 2000.0, 4000.0, 6000.0], dtype=np.float32)
        self.timesteps = np.array([1447893000, 1447903800], dtype=np.int64)
```